### PR TITLE
Add demonstration for escapping a container with SELinux

### PR DESCRIPTION
**Pull request by rhatdan**
Demonstration is being presented first time in Devconf.cz 2019.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: running/SELinuxEscape/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# SELinuxEscape
+
+This script demonstrates what would happen if a container was to escape confinement
```
Comment (vrothberg) [inline running/SELinuxEscape/README.md]: nit: missing dot at the end of the sentence.

File: running/SELinuxEscape/podman.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+blue=$(tput setaf 4)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${blue}$1${reset}"
+}
```
Comment (vrothberg) [inline running/SELinuxEscape/podman.sh]: non-blocking idea: we could refactor those helpers into a common file shared across demos (and document it the main README.md).

File: running/SELinuxEscape/podman.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    sudo systemctl restart auditd 2> /dev/null
+    sudo systemctl restart docker
+    chcon -t container_file_t breakout.sh
+    chmod +x breakout.sh
+    clear
+}
+
+audit() {
+    echo ""
+    echo_color "Lets look at the audit log, to see what SELinux is reporting"
```
Comment (vrothberg) [inline running/SELinuxEscape/podman.sh]: nit: no comma needed.

File: running/SELinuxEscape/podman.sh (hunk 2 above)
Comment (rhatdan) [reply running/SELinuxEscape/podman.sh]: Yes, I plan on doing that next.

File: running/SELinuxEscape/podman.sh (hunk 3 above)
Comment (rhatdan) [reply running/SELinuxEscape/podman.sh]: Fixed

File: running/SELinuxEscape/README.md (hunk 1 above)
Comment (rhatdan) [reply running/SELinuxEscape/README.md]: Fixed

**Comment by rhatdan** [comment]
Ok I am going to merge.  If @TomSweeneyRedHat has any issues, we can fix them in a new PR>

**Comment by TomSweeneyRedHat** [comment]
sorry, didn't see this before now.  Will dive in.

File: running/SELinuxEscape/breakout.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${blue}$1${reset}"
+}
+
```
Comment (TomSweeneyRedHat) [inline running/SELinuxEscape/breakout.sh]: Hmm, I wonder if we could create a common include file for constants and functions like these.  I think so.

Comment (TomSweeneyRedHat) [reply running/SELinuxEscape/breakout.sh]: But then again, one more thing to drag into place, maybe we shouldn't.

File: running/SELinuxEscape/podman.sh (hunk 2 above)
Comment (TomSweeneyRedHat) [reply running/SELinuxEscape/podman.sh]: Honest, I didn't see these comments before my prior!  Poor @vrothberg, here only a few months and already he's starting to think like a Sweeney....

**Pull request merged**